Services.Test/Storage/StorageRecordsTest.cs

```diff
@@ -304,6 +376,26 @@ public void ItTriesToUnlockRecord()
             Assert.True(unlockTask.Result);
```

Comment:
a better test would be checking what the method does, e.g. it calls `this.docDb.DeleteAsync` 5 times

Reply:
Fixed. Updated the test to verify that this.docDb.UpsertAsync is called once.